src/nmodl/deriv.cpp

```diff
@@ -792,12 +776,12 @@ void matchmassage(nderiv)
 		s = SYM(q);
 		ITERATE(q1, deriv_state_list) {
 			if (SYM(q1) == s) {
-				delete(q1);
+				dlete(q1);
```

Comment:
```suggestion
				remove(q1);
```
Looks less like a typo 😄

Reply:
Haha, thanks. Was a quick test that got in unshaped.